l3kernel/l3int.dtx

```diff
@@ -297,7 +297,7 @@
 % \begin{function}[updated = 2011-10-22]
 %   {
 %     \int_set:Nn, \int_set:cn, \int_set:NV, \int_set:cV,
-%     \int_gset:Nn, \int_gset:cn, \int_gset:MV, \int_gset:cV
+%     \int_gset:Nn, \int_gset:cn, \int_gset:NV, \int_gset:cV
```

Comment:
This is already fixed in https://github.com/latex3/latex3/commit/20fa9ac27aa5439fa5f5f6cfc913ec67efcaeeba and has been released starting from release 2024-12-09.

Your `main` branch is 62 commits behind `latex3/latex3:main`:
https://github.com/dbitouze/latex3/compare/main...latex3%3Alatex3%3Amain

Reply:
Ooops! Sorry...